src/server/pkg/util/util.go

```diff
@@ -48,3 +52,33 @@ func GetResourceListFromPipeline(pipelineInfo *pps.PipelineInfo) (*api.ResourceL
 	}
 	return &result, nil
 }
+
+// LookupUser is a reimplementation of user.Lookup that doesn't require cgo.
+func LookupUser(name string) (_ *user.User, retErr error) {
+	passwd, err := os.Open("/etc/passwd")
```

Comment:
Are we making certain assumptions about the user container here?  For instance, what if the container doesn't have `/etc/passwd`?

Reply:
I suppose we are making some assumptions here... I think it's a pretty safe assumption since I'm pretty sure this is how libc does it and I don't think it's really possible to have a linux distribution that libc doesn't work on... but it should be ez enough to handle a not found error here so I'll just do that.